src/skip-to-content/skip-to-content.scss

```diff
@@ -19,7 +19,8 @@
   transform: translateX(-50%) translateY(-170%);
   transition: background-color $iui-speed-fast ease-in-out;
   @media (prefers-reduced-motion: no-preference) {
-    transition: background-color $iui-speed-fast ease-in-out, transform $iui-speed-slow ease-in-out, box-shadow $iui-speed-fast ease-in-out;
+    transition: background-color $iui-speed-fast ease-in-out, transform $iui-speed-slow ease-in-out,
```

Comment:
Why it was changed?

Reply:
It looks like a new line was added after the transform transition.  I didn't do this manually, maybe it was the linter?

Reply:
If I revert this, I get:

![Screen Shot 2022-02-22 at 1 45 40 PM](https://user-images.githubusercontent.com/849817/155198360-61cf3c1f-6b73-46cb-a017-adf7c65057ac.png)

> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -
husky > pre-commit (node v12.22.7)
⚠ Detected incorrect braces with only single value: `*.{scss}`. Reformatted as: `*.scss`
[STARTED] Preparing...
[SUCCESS] Preparing...
[STARTED] Running tasks...
[STARTED] Running tasks for *.scss
[STARTED] Running tasks for *.{scss,html}
[STARTED] Running tasks for *.{scss,html,js}
[STARTED] prettier --write
[STARTED] stylelint --fix
[STARTED] yarn lint:copyright --fix
[SUCCESS] prettier --write
[SUCCESS] Running tasks for *.scss
[SUCCESS] yarn lint:copyright --fix
[SUCCESS] Running tasks for *.{scss,html,js}
[SUCCESS] stylelint --fix
[SUCCESS] Running tasks for *.{scss,html}
[SUCCESS] Running tasks...
[STARTED] Applying modifications...
[FAILED] Prevented an empty git commit!
[STARTED] Reverting to original state because of errors...
[SUCCESS] Reverting to original state because of errors...
[STARTED] Cleaning up...
[SUCCESS] Cleaning up...
  ⚠ lint-staged prevented an empty git commit.
  Use the --allow-empty option to continue, or check your task configuration
husky > pre-commit hook failed (add --no-verify to bypass)

Reply:
Don't revert. I was just curious why linter/prettier changed it, I thought we fixed this.